src/Apps/Artwork/Components/PricingContextModal.tsx

```diff
@@ -36,12 +36,14 @@ export class PricingContextModal extends React.Component<State> {
   }
 
   render() {
+    const linkHref =
+      "https://www.artsy.net/article/artsy-editorial-artworks-prices"
```

Comment:
should we make this a configuration? or even maybe a prop passed to this component? 

Artsy article urls are technically editable, i _think_ we would still support old link but maybe having it as a prop and have the consumer set it may make more sense. @ds300 what do you think?

Reply:
I think this is as good a place as any to specify it. The only reason to parameterize it would be if we decide to use the Modal in more than one place or have it point to different URLs in different contexts.

One suggestion I would make would be to remove the protocol and the host from the url so it works on staging and local.